### ISSUE TITLE
Question about rendering

### ISSUE THREAD

**Issue by cfchi** [VITA-Group/LightGaussian#8]
Hi, thanks for sharing your great work. 
About rendering, I'd like to ask about the generalizability of your work.
According to code in render_video.py

![image](https://github.com/VITA-Group/LightGaussian/assets/118201630/b4cec91d-ae83-4cca-a5ee-f1ba8c696859)

If I use command  'python render_video.py'  without --video, then the operation of rasterization is the same as that of 3DGS, right?
Is it correct that if I only do inference I can render it offline? 
In other words, for inference, the calculation of important scores is unnecessary, and there is no additional calculation comparing with 3DGS.

**Comment by Kevin-2017** [comment]
Thanks for your interest in our work! Yeah, the inference is the same as 3DGS and the important score is only calculated during training.

**Comment by cfchi** [comment]
Thank you for your prompt reply.  I am very impressed by the results you have been able to achieve.
